int32 light_id
string type
string name
string modelid
string swvesion
rocon_hue/HueState state

